src/test/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerMaintainLinkCaseTest.java

```diff
@@ -59,26 +69,31 @@ void shouldNotConfigureMaintainLinkCaseIfFeatureFlagFalse() {
     }
 
     @Test
-    void shouldAdd2LinksToCase() {
+    void shouldSuccessfullyStayTheCase() {
         //Given
-        CicCase cicCase = CicCase.builder()
-            .linkCaseNumber(new CaseLink())
-            .linkCaseReason(LinkCaseReason.CASE_CONSOLIDATED)
-            .build();
-        CaseData caseData = caseData();
+        final CaseData caseData = caseData();
+        CicCase cicCase = new CicCase();
+        cicCase.setSubjectCIC(Set.of(SubjectCIC.SUBJECT));
+        cicCase.setApplicantCIC(Set.of(ApplicantCIC.APPLICANT_CIC));
+        cicCase.setRepresentativeCIC(Set.of(RepresentativeCIC.REPRESENTATIVE));
         caseData.setCicCase(cicCase);
         final CaseDetails<CaseData, State> updatedCaseDetails = new CaseDetails<>();
         final CaseDetails<CaseData, State> beforeDetails = new CaseDetails<>();
         updatedCaseDetails.setData(caseData);
         updatedCaseDetails.setId(TEST_CASE_ID);
         updatedCaseDetails.setCreatedDate(LOCAL_DATE_TIME);
+
         //When
-        AboutToStartOrSubmitResponse<CaseData, State> response1 =
-            caseWorkerMaintainLinkCase.aboutToSubmit(updatedCaseDetails, beforeDetails);
-        SubmittedCallbackResponse submitted = caseWorkerMaintainLinkCase.submitted(updatedCaseDetails, beforeDetails);
+        doNothing().when(caseUnlinkedNotification).sendToSubject(any(CaseData.class), eq(null));
+        doNothing().when(caseUnlinkedNotification).sendToApplicant(any(CaseData.class), eq(null));
+        doNothing().when(caseUnlinkedNotification).sendToRepresentative(any(CaseData.class), eq(null));
+
+        SubmittedCallbackResponse response =
+            caseWorkerMaintainLinkCase.linkUpdated(updatedCaseDetails, beforeDetails);
 
         //Then
-        assertThat(response1).isNotNull();
-        assertThat(submitted).isNotNull();
+        assertThat(response).isNotNull();
```

Comment:
can we assert the response code or something more here?

Reply:
Asserted the confirmation message